src/charm.py

```diff
@@ -28,7 +28,7 @@
 logger = logging.getLogger(__name__)
 
 CONFIG_PATH = "/snap/srsran/current/config"
-WAIT_FOR_UE_IP_TIMEOUT = 20
+WAIT_FOR_UE_IP_TIMEOUT = 80
```

Comment:
Any idea why the previous timeout is not enough anymore?

Reply:
I've never experienced such a short attaching time, every time I did it took ~40-50 seconds. Idk why the timeout was so short, never used this charm after the refactor. Maybe @gruyaume can tell.